### DISCUSSION TITLE
[FEATURE REQUEST]: Add wttrbar to waybar config

### DISCUSSION THREAD

**Discussion by gnussbaum67** [prasanthrangan/hyprdots#1626] [Ideas]
I was wondering if you could add wttrbar (weather) to the waybar config?

https://github.com/bjesus/wttrbar

I understand if you can't, but it would be a nice option to have.

**Comment by dieBakterie** [comment]
I guess they'll not add this to avoid the dep/extra package, that's why they've the keybinds_hint.sh to avoid hyprkeys(extra package). If you want i've an weather waybar module and an python script for this.

- [weather.jsonc](https://github.com/user-attachments/files/16028544/weather.jsonc)
- [weather.txt](https://github.com/user-attachments/files/16028587/weather.txt) That's a python script just rename it to `weather.py`

Hope this helps you don't really know why they haven't added it this way. There are multiple PR for this. One from me, moguay and some more.
